docs/defects-api.yml

```diff
@@ -198,6 +260,81 @@ components:
       type: array
       items:
         $ref: "#/components/schemas/imNumber"
+
+    ivaDefects:
+      type: array
+      items:
+        $ref: "#/components/schemas/ivaDefect"
+
+    ivaDefect:
+      type: object
+      properties:
+        sectionNumber:
+          type: string
+        sectionDescription:
+          type: string
+        vehicleType:
+          $ref: "#/components/schemas/vehicleType"
+        manualType:
+          $ref: "#/components/schemas/euVehicleCategory"
+        requiredStandards:
+          type: array
+          items:
+            $ref: '#/components/schemas/requiredStandard'
+
+    requiredStandard:
+      properties:
+        rsNumber:
+          type: integer
+        requiredStandard:
+          type: string
+        refCalculation:
+          type: number
+        additionalInfo:
+          type: string
+        basic:
+          type: boolean
+        normal:
+          type: boolean  
+  
+    vehicleType:
+      type: string
+      enum:
+        - car
+        - lgv
+        - motorcycle
+        - psv
+        - hgv
+        - trl
+    
+    euVehicleCategory:
+      type: string
+      enum:
+        - m1
+        - m2
+        - m3
+        - n1
+        - n2
+        - n3
+        - o1
+        - o2
+        - o3
+        - o4
+        - l1e-a
+        - l1e
+        - l2e
+        - l3e
+        - l4e
+        - l5e
+        - l6e
+        - l7e
```

Comment:
The endpoint will never be able to be provided with these, if you can't do IVA's against a vehicle with these u can't defect